ext/ruby_binlog.cpp

```diff
@@ -411,6 +423,7 @@ struct Client {
     rb_define_method(rb_cBinlogClient, "position=", __F(&set_position2), 1);
     rb_define_method(rb_cBinlogClient, "get_position", __F(&get_position), -1);
     rb_define_method(rb_cBinlogClient, "position", __F(&get_position2), 0);
+    rb_define_method(rb_cBinlogClient, "set_ssl_ca", __F(&set_ssl_ca), 1);
```

Comment:
You're right. From the stand point of API, we should accept hash object for the second params of constructor. I will add that.